packages/api-specification/transform-type-info.js

```diff
@@ -150,7 +150,7 @@ const documentClass = (className) => {
           h5('Returns'),
           dl([
             dt(formatType(d.match.type), {class: 'code return-value'}),
-            dd(formatComment(d.match.comment))
+            dd(d.match.comment ? d.match.comment.returns : '')
```

Comment:
I wasn't sure what the 'schema' for comments was, which is why I pushed everything through the `formatComment` function.

Would it work if you just fixed the mistake I made in the function ...

comment.return => comment.returns